### DISCUSSION TITLE
可以在设置里面增加一个关闭托盘图标的功能吗？

### DISCUSSION THREAD

**Discussion by huaouo** [he3-app/he3-feedback#180] [General]
托盘图标上只有 Open He3 和 Quit 两个按钮，感觉必要性不强。呼出 He3 用快捷键更方便。

**Comment by lyzhang1999** [comment]
感谢您的建议，我已经创建了新的 issues：https://github.com/he3-app/he3-feedback/issues/182，您可以关注一下。
